src/System.Private.CoreLib/src/System/Threading/RuntimeThread.cs

```diff
@@ -2,33 +2,123 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
+using System.Diagnostics;
+using System.Runtime;
+using Microsoft.Win32.SafeHandles;
 
 namespace System.Threading
```

Comment:
Since an instance of RuntimeThread needs to be I.R.A.RuntimeThread, which Thread derives from, I.R.A.RuntimeThread would need to derive from the temporary S.T.RuntimeThread to forward instance members without an extra indirect and the main functionality would need to temporarily remain in S.T.RuntimeThread. Or is there another way? Wouldn't it be easier to just target I.R.A.RuntimeThread directly from your changes?